static/app/components/eventOrGroupHeader.tsx

```diff
@@ -103,13 +105,16 @@ function EventOrGroupHeader({
       );
     }
 
+    // If we have passed in a custom event ID, use it; otherwise use default
+    const finalEventId = eventId ?? eventID;
+
     return (
       <TitleWithLink
         {...commonEleProps}
         to={{
           pathname: `/organizations/${organization.slug}/issues/${
             eventID ? groupID : id
-          }/${eventID ? `events/${eventID}/` : ''}`,
```

Comment:
before, were we generating links with all the parts included, like  `/issues/{$groupID}/event/${eventID}/`?

or were the links only the first bit like `/issues/${id}/`?

Reply:
the link didn't include `/event/${eventID}/`; it wasn't getting the `eventID` at all in our situation

Reply:
something with casting the issue as en `Event` was going wrong

Reply:
it seems like before, if we passed a `Group` in, then the extra `/event/xyz/` part of the url would not appear here. 

We should be careful not to break other callsites. However the logic for what's rendered is unnecessarily complex imo

Reply:
kk, cool. as long as that `finalEventId` ternary check is still there i think it's good.